python/sglang/lang/backend/anthropic.py

```diff
@@ -67,6 +94,20 @@ def generate_stream(
         else:
             system = ""
 
+        debug_request_id = str(uuid.uuid4())
+        debug_obj = s.log_debug(
+            [
+                {
+                    "id": debug_request_id,
+                    "requestPrompt": str(
+                        [{"role": "system", "content": system}] + messages
+                    ),
+                    "requestTimestamp": datetime.now().isoformat(),
+                    "requestMetadata": sampling_params.to_anthropic_kwargs(),
+                }
+            ]
+        )
```

Comment:
This is not efficient enough. When debug is turned off, you still run the code to construct the argument, which takes some time. Please minimize the overhead and do not construct any objects when debug is turned off.